torch_tvm/custom_tvm_ops/cpp/relay/quantize_init.cc

```diff
@@ -35,6 +38,16 @@ RELAY_REGISTER_OP("nn.quantize_data_int8_quantize")
   .add_type_rel("DataInt8Quantization", DataInt8QuantizationRel);
 
 
+RELAY_REGISTER_OP("nn.quantize_data_int8_row_offset")
+.describe(R"code(dynamic row offset calculation of quantized data.
+- **data**: (M, N)
+)code" TVM_ADD_FILELINE)
+  .set_num_inputs(1)
+  .add_argument("data", "Tensor", "Quantized input tensor.")
+  .set_support_level(10)
+  .add_type_rel("DataInt8RowOffset", DataInt8RowOffsetRel);
```

Comment:
In terms of return multiple tensors, can you check "vision.get_valid_counts" in tvm repo?

Reply:
@llyfacebook, so I checked that. The difference really is because of `@autotvm.register_topi_compute(data_int8_quantize, 'cpu', ['direct'])`. It is the `register_topi_compute` of autotvm that breaks. If we just go with generic_func like how they do it for get_valid_counts, it would be ok, but I did not try that. Maybe we can do that, although I am not too sure if that is going to buy us any advantage at the moment, because it seems unlikely that we can fuse `data quantize` and `row offset calc` as of now. (Mainly because right now you cannot inline primary outputs of an op)